pom.xml

```diff
@@ -16,11 +16,11 @@
     </modules>
 
     <properties>
-        <java.version>21</java.version>
-        <maven.compiler.source>${java.version}</maven.compiler.source>
-        <maven.compiler.target>${java.version}</maven.compiler.target>
-        <maven.compiler.release>${java.version}</maven.compiler.release>
-        <javafx.version>21</javafx.version>
+        <java.version>21.0.1</java.version>
```

Comment:
roll back this change, but rename to `java.major.version`?

Reply:
`java.version` is used in scene builder to show the java information in about dialog

Reply:
ok